modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -135,33 +141,107 @@ This expands the dialog vertically, as follows:
 [#configure-ldap-dialog-enable-ldap-user-auth-field]
 image::manage-security/configureLdapEnableLdapUserAuthField.png[,400,align=left]
 +
-This provides three options whereby usernames can be mapped on the LDAP server.
+This provides three options whereby usernames can be mapped to LDAP distinguished names (https://ldap.com/ldap-dns-and-rdns/[DN^]).
 The default option is *Template*.
-An appropriate template should be entered into the *Template* editable text field.
+An appropriate mapping should be entered into the *Template* editable text field: this avoids the need to request the LDAP server to provide the username-to-DN mapping.
+The required format is indicated by the placeholder text, within the field.
+The `%u` is replaced by the username used at login.
++
+For example, based on the placeholder, if the user logs in as `exampleUser`, the user is mapped to the DN `cn=exampleUser,dc=example,dc=com`.
```

Comment:
in placeholder the different template is shown. Correct example: if template is "cn=%u,dc=example,dc=com" and user logs in as `exampleUser`, the user is mapped to the DN `cn=exampleUser,dc=example,dc=com`

Reply:
I have modified as described.